GitHubConfiguration.ps1

```diff
@@ -180,6 +184,8 @@ function Set-GitHubConfiguration
 
         [switch] $LogProcessId,
 
+        [switch] $LogRequestBody,
```

Comment:
Instead of adding a new switch, is it possible to just use the existing `-Debug` switch? It would also be better for the module if it made better use of the various output streams so that users could redirect or suppress different levels of information.

Reply:
This is a great suggestion at a high level, but becomes more complicated when we start dealing with jobs that are executing asynchronously.  I found [this interesting post](https://social.technet.microsoft.com/forums/windowsserver/en-US/696ceda1-ea2a-43ba-b648-9ce058db210b/how-to-debug-a-job-created-using-startjob) about debugging an executing Job, but it looks like this would end up complicating the code quite a bit further.  I think for now, inspection via an optional log/verbose output helps satisfy this direct need, and we can look at adding `Write-Log -Level Debug` messages in other places on an as-is needed basis in the future.